src/components/MetricsPane/charts/SampleEventBleachingSeverityPlot.jsx

```diff
@@ -19,39 +19,41 @@ export const SampleEventBleachingSeverityPlot = ({ coloniesBleachedData }) => {
     percent_pale_avg: percentPale,
     sample_unit_count: sampleUnitCount,
   } = coloniesBleachedData
-  const isBleachingSeverityDataAvailable =
-    !!percent100 &&
-    !!percent20 &&
-    !!percent50 &&
-    !!percent80 &&
-    !!percentDead &&
-    !!percentNormal &&
-    !!percentPale &&
-    !!countTotalAvg
+
+  const isBleachingSeverityDataAvailable = [
+    percent100,
+    percent20,
+    percent50,
+    percent80,
+    percentDead,
+    percentNormal,
+    percentPale,
+    countTotalAvg,
+  ].every((value) => value !== null && value !== undefined)
```

Comment:
I wonder if this is worth turning into a utility function. that might be overkill. I dont have an opinion.